src/frontend/src/components/irregularity.ts

```diff
@@ -0,0 +1,48 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon, closeIcon } from "./icons";
+
+interface irregularityProps {
+  message: string | TemplateResult;
+  additionalClasses?: string[];
+  type?: "warning" | "info" | "error";
+  closeFn?: () => void;
+}
+
+export const irregularity = ({
+  message,
+  additionalClasses = [],
+  type = "error",
```

Comment:
I think type is a reserved keyword for typescript, this might cause issues

Reply:
I will rename it to error type then, (The linter did not complain about it though)